### PR TITLE
[Doc] Added John Snow Lab's NLU reference to community integration page

### PR DESCRIPTION
**Pull request by C-K-Loan**
Added logo, text and reference of Modin usage in NLU

### CODE REVIEW

File: doc/source/ray-libraries.rst [hunk 1]
```diff
@@ -94,6 +105,9 @@ Spacy |spacy|
 -------------
 spaCy is a library for advanced Natural Language Processing in Python and Cython. It's built on the very latest research, and was designed from day one to be used in real products.
 
+
+
+
```
Comment (richardliaw) [inline doc/source/ray-libraries.rst:110]:
```suggestion
```

File: doc/source/ray-libraries.rst [hunk 2]
```diff
@@ -41,6 +41,17 @@ Horovod is a distributed deep learning training framework for TensorFlow, Keras,
...
+-----------------------------------
+
+
+The power of 350+ pre-trained NLP models, 100+ Word Embeddings, 50+ Sentence Embeddings, and 50+ Classifiers in 46 languages with 1 line of Python code.
+
+[`Link to integration <https://nlu.johnsnowlabs.com/docs/en/predict_api#modin-dataframe>`__]
+
+
+
+
```
Comment (richardliaw) [inline doc/source/ray-libraries.rst:54]:
```suggestion
The power of 350+ pre-trained NLP models, 100+ Word Embeddings, 50+ Sentence Embeddings, and 50+ Classifiers in 46 languages with 1 line of Python code.

[`Link to integration <https://nlu.johnsnowlabs.com/docs/en/predict_api#modin-dataframe>`__]

```

**Review by richardliaw — APPROVED** [review]

File: doc/source/ray-libraries.rst [hunk 3]
```diff
@@ -41,6 +41,12 @@ Horovod is a distributed deep learning training framework for TensorFlow, Keras,
 
 [`Link to integration <https://horovod.readthedocs.io/en/stable/ray_include.html>`__]
 
+John Snow Labs' NLU |NLU|
```
Comment (ericl) [inline doc/source/ray-libraries.rst:44]: Could you sort this alphabetically to after 'I'?

File: doc/source/ray-libraries.rst [hunk 4]
```diff
@@ -143,3 +149,7 @@ spaCy is a library for advanced Natural Language Processing in Python and Cython
 .. |spacy| image:: images/spacy.png
     :class: inline-figure
     :height: 30
+    
+.. |nlu| image:: images/nlu.png
```
Comment (ericl) [inline doc/source/ray-libraries.rst:153]: Btw I noticed the logo has a bunch of whitespace, it would be better to trim that before uploading.

**Pull request merged**